### DISCUSSION TITLE
关于B站的开屏广告问题

### DISCUSSION THREAD

**Discussion by fansangg** [gkd-kit/gkd#684] [Q&A]
B站的开屏广告是在MainActivity里的，感觉是ViewTree节点太多了导致查询慢，大概要2秒左右才会跳过
我的规则是[text^=“跳”]
有什么优化的办法吗？
@lisonge

**Comment by lisonge** [comment]
1. 使用 fastQuery 优化查询
2. 其它所有开启的规则不能是无法开启 fastQuery 的，否则将继续阻塞导致查询慢
